roles/robots_txt/tasks/main.yml

```diff
@@ -1,11 +1,20 @@
 ---
 # Add cnx robots.txt
 
-- name: Add robots.txt
+- name: Add top level robots.txt
```

Comment:
"top level" and it's a directory below the dir which holds webview's robots.txt. Not a biggie but maybe "Add webview robots.txt"?

Reply:
Actually, they're both top level when served via http- one from cnx.org, one from archive.cnx.org. Confusingly, the cnx.org one ("top level" for domain) is served from lower down the filesystem tree, while the archive.cnx.org one is (here) now being served from the nginx webroot. Sane? Not really. I considered adding an 'archive' subdir, for the one file, but that would have required even more changes (an extra stanza in nginx config as well)

Reply:
I was just suggesting clarifying the comment.